.devcontainer/Dockerfile.console

```diff
@@ -0,0 +1,19 @@
+FROM openshift/origin-cli:latest AS cli
+FROM quay.io/coreos/tectonic-console-builder:v23 AS build
```

Comment:
Any reason not to use `quay.io/openshift/origin-console` instead? Then you won't have to build console below.